includes/meta.php

```diff
@@ -157,6 +213,22 @@ function save_post( $post_id ) {
 
```

Comment:
comparing $new_meta and $existing_meta when a new tag is passed, the array_map() intval it just checks against a zero. it works (because the arrays still don't match) but wanted to note that.